BangazonAPI/BangazonAPI/Controllers/CustomerController.cs

```diff
@@ -0,0 +1,330 @@
+                    {
+                        command = $@"{customerColumn}
+                                      {customerTable}";
+                    }
+
+                    //If statement for 'q' string, user sets q='whatever' and find 'FirstName/LastName' that is LIKE what user sets 'q=';
+                    //Future reference, don't forget to add '%' outside {q}
+                    if (q != null)
+                    {
+                        command += $" WHERE c.FirstName LIKE '{q}%' OR c.LastName LIKE '{q}%'";
+                    }
+                    //Another query string, doing the same thing as product except w/ payments
+                    if (include == "payments")
+                    {
+
+                        string paymentColumn = @",pm.Id AS 'Payment Id',
+                                                  pm.Name AS 'Payment Name',
+                                                  pm.AcctNumber AS 'Payment Account Number'";
+                        string paymentTable = @"Join PaymentType pm on c.Id = pm.CustomerId";
+                        //Adding the strings together to show customer and payment
+                        command = $@"{customerColumn}
+                                     {paymentColumn}
+                                     {customerTable}
+                                     {paymentTable}";
+                    }
+
+
+
+
+
+                    cmd.CommandText = command;
+                    SqlDataReader reader = cmd.ExecuteReader();
+                    List<Customer> customers = new List<Customer>();
+
+                    while (reader.Read())
+                    {
+                        Customer attempt = new Customer
+                        //Getting the information of the customer - FUTURE JOSH MAKE SURE YOU REFERENCE WHAT YOU SET THE COLUM NAME 'AS' --
+                        {
+                            Id = reader.GetInt32(reader.GetOrdinal("Customer Id")),
+                            FirstName = reader.GetString(reader.GetOrdinal("Customer First Name")),
+                            LastName = reader.GetString(reader.GetOrdinal("Customer Last Name"))
+                        };
+                        
+                        //Getting the information of the products if include == 'products'
+                        if(include == "products")
+                        {
+                            Product currentProduct = new Product
+                            {
+                                Id = reader.GetInt32(reader.GetOrdinal("Product Id")),
+                                Price = reader.GetInt32(reader.GetOrdinal("Product Price")),
+                                Title = reader.GetString(reader.GetOrdinal("Product Title")),
+                                Description = reader.GetString(reader.GetOrdinal("Product Description")),
+                                Quantity = reader.GetInt32(reader.GetOrdinal("Product Quantity"))
```

Comment:
When we include products, they have a `CustomerId` of 0 (because we're not selecting that property in our SQL query). That makes it hard to figure out if the data is correct, let's add that FK back in